### ISSUE TITLE
Faction Discovery: Empire conflict

### ISSUE THREAD

**Issue by OrionFive** [OrionFive/Hospitality#397]
Hey, to fix incompatibilities with Empire, do the following:

When you ( I assume cycle through each factiondef and check to see if the faction exists) if the defname is equal to one of the following, skip it: “PColony”, “PColonyMedieval”, “PColonySpacer”, “PColonyIndustrial”, “PColonyTribal”.

The latter four are being phased out, but I am giving time for players to update Empire so it doesn’t break their games.

**Comment by OmanMkII** [comment]
I've found a minor conflict with a set of Empire guests currently in my colony (accidentally overwrote my permadeath save and can't reproduce, sorry).

When I had a group (Empire faction) staying in my colony I also accepted a quest to house a member of the Empire and one guest. I believe one of the guests had the same name as the person I was meant to house and/or their protector which triggered failing the quest. I've been unable to reproduce it but I believe it was conflicting names since one of the guests was the target of the notification about their defecting.
